src/query/shorthand.ts

```diff
@@ -1,4 +1,7 @@
 import {Type} from 'vega-lite/src/type';
+import {filter} from 'vega-lite/src/compile/data/filter';
```

Comment:
You supposed to move it to `src/filter`  https://github.com/vega/vega-lite/issues/1491